src/DGtal/geometry/curves/AlphaThickSegmentComputer.h

```diff
@@ -130,89 +130,89 @@ class AlphaThickSegmentComputer
 
   // ----------------------- public types --------------------------------------
   BOOST_STATIC_ASSERT(( TInputPoint::dimension == 2 ));
-  BOOST_CONCEPT_ASSERT((boost_concepts::ReadableIterator<TConstIterator>)); 
- 
+  BOOST_CONCEPT_ASSERT((boost_concepts::ReadableIterator<TConstIterator>));
+
 public:
   /**
    * Type of input point.
    **/
   typedef TInputPoint InputPoint;
-  
+
   /**
    * The container type of Input Point
    **/
   typedef std::vector< InputPoint > InputPointContainer;
   typedef typename InputPointContainer::size_type Size;
   typedef typename InputPointContainer::const_iterator ContainerConstIterator;
   typedef typename DGtal::functions::Hull2D::ThicknessDefinition ThicknessDef;
-  
+
   typedef typename InputPointContainer::iterator Iterator;
   typedef TConstIterator ConstIterator;
-  typedef ParallelStrip< SpaceND< 2,  DGtal::int32_t > ,true,true> Primitive;  
-  
+  typedef ParallelStrip< SpaceND< 2 > ,true,true> Primitive;  
```

Comment:
It works when hardcoding `ParallelStrip<SpaceND< 2, int32_t>, ...>` because his `operator()` has a surcharge for `Point` and `RealPoint` (always double) from `SpaceND`.